src/i18n/en-us/index.js

```diff
@@ -190,6 +192,14 @@ export default {
         },
       },
       treasuries: {
+        settings: {
+          burnable: 'If enabled, the token manager can burn their own token',
```

Comment:
only the token manager can burn the tokens or any owner could do that?

Reply:
It requires the signature of the manager. You can look [here at the code](https://github.com/telosnetwork/telos-decide/blob/master/contracts/decide/src/treasury.cpp#L208). It says:

```c++
    //open treasuries table, get treasury
    treasuries_table treasuries(get_self(), get_self().value);
    auto& trs = treasuries.get(quantity.symbol.code().raw(), "treasury not found");

    //authenticate
    require_auth(trs.manager);

```